tests/lora/test_lora_layers_old_backend.py

```diff
@@ -2297,21 +2301,36 @@ def test_sdxl_1_0_last_ben(self):
         self.assertTrue(np.allclose(images, expected, atol=1e-3))
 
     def test_sdxl_1_0_fuse_unfuse_all(self):
-        pipe = DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16)
+        pipe = DiffusionPipeline.from_pretrained(
+            "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.bfloat16
```

Comment:
Does this break with `float16` with our test runner? @younesbelkada could you check this?